content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -54,6 +54,14 @@ stateful tasks.
 
 {{< figure src="/images/docs/scheduling-framework-extensions.png" title="scheduling framework extension points" class="diagram-large">}}
 
+### PreEnqueue {#pre-enqueue}
+
+These plugins are called prior to adding Pods to the internal active queue, where Pods are marked as
```

Comment:
do we have any docs that refer to the internal active queue and unschedulable list? if so, I think it is worth linking here.

Reply:
yes, we have one here: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-scheduling/scheduler_queues.md. Linked.